app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -156,11 +158,57 @@ private void initView() {
         adapter = new SessionsAdapter(getContext());
         binding.recyclerView.setAdapter(adapter);
 
+        GestureDetector gestureDetector = new GestureDetector(getContext(), new GestureDetector.OnGestureListener() {
+            private boolean ignoreMotionEventTillDown = false;
+
+            @Override
+            public boolean onDown(MotionEvent motionEvent) {
+                ignoreMotionEventTillDown = true;
```

Comment:
Once `ignoreMotionEventTillDown` is set true here, this flag will be never changed.
Is it okay?